src/BenchmarkDotNet.Core/Diagnosers/CompositeDiagnoser.cs

```diff
@@ -19,13 +19,13 @@ public CompositeDiagnoser(params IDiagnoser[] diagnosers)
 
         public IColumnProvider GetColumnProvider() 
             => new CompositeColumnProvider(diagnosers.Select(d => d.GetColumnProvider()).ToArray());
-
+#if !UAP
```

Comment:
is the `Process` class not available for UAP? If so I have one idea how we could get rid of most of the `#if`. Maybe we could define a class that is called Process and give it the same api but just throw NotImplementedException for every method? Then the rest of the BDN code could remain untouched? Sth like

```cs
#if UWP
namespace System.Diagnostics
{
   public class Process
   {

   }
}
#endif
```